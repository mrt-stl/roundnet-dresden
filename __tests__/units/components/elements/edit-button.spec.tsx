import { create } from "react-test-renderer"
import EditButton from "../../../../components/elements/edit-button"

/**
 * This is a snapshot test. A snapshot test is taking when the component
 * doesn't change that often. Avoid snapshot testing if the component is
 * about to change often. Marker component should not change this often.
 */

const testDocId = "test doc id"

describe("Marker component", () => {
    test("Matches the snapshot", () => {
        const editBtn = create(
            <EditButton
                docId={testDocId} />
        )
        expect(editBtn.toJSON()).toMatchSnapshot()
    })
})