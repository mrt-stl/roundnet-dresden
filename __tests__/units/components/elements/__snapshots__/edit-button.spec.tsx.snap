// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marker component Matches the snapshot 1`] = `
<div
  className="edit-button"
  data-wio-id="test doc id"
/>
`;
